app/views/pages/home.html.erb

```diff
@@ -1,2 +1,78 @@
+     <div class="container">
+       <h1>Credify <strong> Unlock your spending potential! </strong> with <strong>Thousands of credit cards for rent</strong>!</h1>
+       <p>Change your life, Use Credify</p>
+       <%= link_to "Sign up", "#", class: "btn btn-gradient" %>
+     </div>
+   </div>
+
+
+
+<div class= "container py-5">
+
+      <H2><STrong>Find a Card that Suits your needs Today!!!</STrong></H2>
+
+<div class= "row justify-content-center py-3">
+    <div class= "col-12 col-lg-6">
+      <div class="credit-card">
+        <img src="https://www.costco.ca/wcsstore/CostcoCABCCatalogAssetStore/feature-pages/17w1217-membership-card-executive.png" />
+        <div class="card-trip-infos">
+          <div>
+            <h2>Costco Card</h2>
+            <p>Make your trips to Costco worth it!</p>
+          </div>
+          <h2 class="card-trip-pricing">$50</h2>
+          <img src="https://kitt.lewagon.com/placeholder/users/krokrob" class="card-trip-user avatar-bordered" />
+        </div>
+      </div>
+    </div>
+
+    <div class= "col-12 col-lg-6">
+      <div class="credit-card">
+        <img src="https://cdn.corporate.walmart.com/dims4/WMT/4f9552e/2147483647/strip/true/crop/1125x750+0+0/resize/1960x1306!/quality/90/?url=https%3A%2F%2Fcdn.corporate.walmart.com%2Fd2%2Fec%2Fbe9650a94452a421323ffa56a289%2Fwalmart-credit-card.PNG" />
+        <div class="card-trip-infos">
+          <div>
+            <h2>Walmart Card</h2>
+            <p>Groceries that change life!</p>
+          </div>
+          <h2 class="card-trip-pricing">$80</h2>
+          <img src="https://kitt.lewagon.com/placeholder/users/krokrob" class="card-trip-user avatar-bordered" />
+          </div>
+      </div>
+    </div>
+
+     <div class= "col-12 col-lg-6">
+      <div class="credit-card">
+        <img src="https://c1.neweggimages.com/ProductImage/32-703-002CVF~004CVF.JPG" />
+        <div class="card-trip-infos">
+          <div>
+            <h2>Air Canada Card</h2>
+            <p>A new way to fly!</p>
+          </div>
+          <h2 class="card-trip-pricing">$90</h2>
+          <img src="https://kitt.lewagon.com/placeholder/users/krokrob" class="card-trip-user avatar-bordered" />
+          </div>
+         </div>
+      </div>
+
+    <div class= "col-12 col-lg-6">
+      <div class="credit-card">
+        <img src="https://www.storefrontdirect.com/pub/media/catalog/product/cache/8c0cd180e440eae853fa2176b62a6ddf/p/p/ppx_high_res_generic_1.png" />
+        <div class="card-trip-infos">
+          <div>
+            <h2>Pharmaprix Card</h2>
+            <p>A new pharmacy shoping experience!</p>
+          </div>
+          <h2 class="card-trip-pricing">$100</h2>
+          <img src="https://kitt.lewagon.com/placeholder/users/krokrob" class="card-trip-user avatar-bordered" />
+          </div>
+      </div>
+    </div>
+</div>
+
+    </div>
+
+
+    </div>
```

Comment:
I think this div is extra? This is in another colour according to github. It's worth to check that.